ups/obs_cfht.table

```diff
@@ -1,4 +1,3 @@
-setupRequired(daf_persistence)
```

Comment:
There is a comment in this file about DM-30891 -- that ticket is now closed so maybe this can be resolved (see eg lsst/obs_lsst#380)

Reply:
Is this comment about the meas_algorithms and meas_astrom mentions below?

Reply:
Yes, I was just responding to your comment above. Somehow github doesn't always render them together?